Lib/fontbakery/profiles/googlefonts.py

```diff
@@ -3791,6 +3792,83 @@ def com_google_fonts_check_name_family_and_style_max_length(ttFont):
     yield PASS, "All name entries are good."
 
 
+@check(
+    id='com.google.fonts/check/has_unacceptable_control_chars',
+    conditions=['are_ttf']
+)
+def com_google_fonts_check_has_unacceptable_control_chars(ttFonts):
```

Comment:
It seems to me that this check does not really need to deal with all font files at once. It could be simpler by running on single files.

On the other hand, the benefit of this approach would be to report only once per family. And typically the glyph-set is the same across a family. So it is not really bad to implement it as a family check.

Reply:
Let me know if there is something actionable for me here.  Happy to remove the mult-font file approach if you would prefer that.

Reply:
no, I think for now it is OK. But we should use the "check/family/" check-id format if we'll keep it as a family-wide check